app/styles/tasks.css

```diff
@@ -202,6 +206,58 @@ input[type=range]:focus::-ms-fill-upper {
   margin-top: 10px;
 }
 
+/* modal styles */
+
+.overlay {
+  background: var(--opaque-black);
+  position: fixed;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 1;
+}
+
+.modal {
+  width: 90%;
+  max-width: 650px;
+  margin: 0 auto;
+  align-self: center;
+  padding: 0.2rem 0.2rem;
+  background: var(--white);
+  color: var(--black);
+  position: relative;
+  top: 40%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 1rem;
+}
+
+.modal-title {
+  font-size: 1.1rem;
+  color: var(--black);
+  margin-bottom: 1rem;
+  font-weight: 600;
+}
+
+.proceed-btn {
+  display: block;
+  width: 70%;
+  margin: .4rem;
+  padding: 0.5rem;
+  background-color: var(--green);
+  border: none;
+  color: white;
```

Comment:
Can you assign the white variable to the color property instead of directing assigning the white color?

Reply:
will do that